app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoScreen.kt

```diff
@@ -131,4 +135,18 @@ class IntroVideoScreen(
       CrashReporter.report(ActivityNotFoundException("Unable to play simple video because no supporting apps were found."))
     }
   }
+
+  @Parcelize
+  data class Key(
+      val registrationEntry: OngoingRegistrationEntry
+  ) : ScreenKey() {
+
+    @IgnoredOnParcel
+    override val analyticsName = "Onboarding Intro Video"
```

Comment:
Can we move this inside the `data class` constructor, please? We are starting to do that to avoid the lint issues when using `IgnoredOnParcel` annotation.

Reply:
Can you link the reference to the issue that you're talking about?